curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646ad836ae56d30140a7b4eb.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 646ad836ae56d30140a7b4eb
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+In this step, you will do 2 things: 
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed